### ISSUE TITLE
ballerina init should enforce package naming rules

### ISSUE THREAD

**Issue by raj-rajaratnam** [ballerina-platform/ballerina-lang#10638]
**Description:**
*ballerina init* allows a package to be named with hyphen in it, and *ballerina push* allows it to be pushed to the repo, but it cannot be used in the import statement. Doc says package name can contain only alpha numeric + period. I think we have to enforce this rule in *ballerina init* as well

**Comment by NatashaWso2** [comment]
@raj-rajaratnam we do validate the package name in _ballerina init_  as below:
![validatepkg](https://user-images.githubusercontent.com/13446645/46589902-8f3bcc00-cacc-11e8-8211-2154141845d0.png)

If a package name contains hyphens we give an error to the user as above.

**Comment by pubudu91** [comment]
@NatashaWso2 Try the following to reproduce
- Run `ballerina init`
- Make a dir with a hyphen in the name and add a Ballerina source file. e.g., `dir-pkg`
- Run `ballerina install dir-pkg`
- Now try to import `dir-pkg` in another source file

**Comment by NatashaWso2** [comment]
@raj-rajaratnam yes this validation is not provided for _ballerina install_ but is enforced on _ballerina init and push_. I'll work on adding this validation to the install command.

**Comment by raj-rajaratnam** [comment]
@NatashaWso2 I'm on v0.981.1 and this validation is not there in ballerina init and push. I guess this is fixed in latest version?

```
Rajs-MacBook-Pro:demo raj$ ballerina init -i
Create Ballerina.toml [yes/y, no/n]: (y) y
Organization name: (raj)
Version: (0.0.1)
Ballerina source [service/s, main/m, finish/f]: (f) s
Package for the service : (no package) package-with-hyphen
Ballerina source [service/s, main/m, finish/f]: (f) f

Ballerina project initialized
```

**Comment by NatashaWso2** [comment]
@raj-rajaratnam yes this is fixed in the latest ballerina version 0.982.0 released.
